lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -156,6 +156,7 @@ module ConfigConstants
 
     # Internal constants.
     module InternalConstants
+      CREDENTIAL_PATH_VAR = 'GOOGLE_APPLICATION_CREDENTIALS'.freeze
```

Comment:
Why have a named constant? Our test is full of literal uses of this value.
If you do choose to keep it, `CREDENTIALS_PATH_ENV_VAR`, for consistency.

Reply:
Because... I sometimes missed the `S` when spelling it. Just wanna reduce such risks. I've renamed it to `CREDENTIALS_PATH_ENV_VAR` instead. Also refactored the tests.